examples/meta/generator/targets/cpp.json

```diff
@@ -0,0 +1,61 @@
+{
+    "Program": "#include <shogun/base/init.h>\n#include <shogun/io/SGIO.h>\n#include <shogun/features/DenseFeatures.h>\n#include <shogun/labels/MulticlassLabels.h>\n#include <shogun/multiclass/KNN.h>\n#include <shogun/io/CSVFile.h>\n#include <shogun/distance/EuclideanDistance.h>\n\n${dependencies}\n\nusing namespace shogun;\n\nint main(int, char*[]) {\n    init_shogun_with_defaults();\n\n$program\n    exit_shogun();\n    return 0;\n}\n",
```

Comment:
Hmm these imports are annoying. We can define import statements in the `"Dependencies"` field, but I don't know where we'd get the path information from (like `"shogun/features/DenseFeatures.h"`). 

The meta-language just knows the name of the classes that are being used, not where they're located. Looks like we need are more complex class scraper that includes that info. Would that be fragile to files being moved around in different releases of shogun? What do you think?